src/pages/Auth/ForgotPassword.js

```diff
@@ -0,0 +1,73 @@
+import { useState } from "react";
+import { showToast } from "utilities";
+import axios from "axios";
+import { useDocumentTitle } from "hooks";
+import { ThreeDots } from "react-loader-spinner";
```

Comment:
Appreciation - great to see you are trying out something new!

Reply:
Thanks a lot!